meta-lmp-bsp/recipes-kernel/linux/linux-lmp_%.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}/${KSHORT_VER}:${THISDIR}/${PN}:"
```

Comment:
What happens if you use PV here instead?

Reply:
@angolini the standard PV for kernel includes SRCPV which is changed each time we bump a particular kernel. Usually patches can be applied without changes to any kernel with the same major/minor version, like 6.6. So this change allows us to use patchsets tuned for different major/minor kernel with the same patch names.

Reply:
Yes, we have a lot of patches for linux-lmp-fslc-imx 6.1 and 6.6, and saving the short version of PV allows us to store different versions of the same patches for different kernel versions in the fine structure like
/linux-lmp/6.1/
/linux-lmp/6.6/
or
/linux-lmp-fslc-imx/6.1/
/linux-lmp-fslc-imx/6.6/